app/assets/stylesheets/_variables.scss

```diff
@@ -147,7 +147,8 @@ $spacers: map-merge((
   'sm': ($spacer * 2),
   'md': ($spacer * 4),
   'lg': ($spacer * 6),
-  'xl': ($spacer * 8)
+  'xl': ($spacer * 8),
+  'header-with-covid-banner': ($spacer * 7.25)
```

Comment:
This should just be a custom class. Typically the variables should be reserved for site-wide stuff